charts/incubator/external-dns/templates/_args.tpl

```diff
@@ -0,0 +1,48 @@
+{{- define "externaldns.args" -}}
+args:
+  {{- with .Values.externaldns.provider }}
+  - --provider={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.zoneidfilter }}
+  - --zone-id-filter={{ . }}
+  {{- end -}}
+  {{- with .Values.externaldns.cloudflareproxied }}
```

Comment:
Can we change those to `camelCase`? (same on values and questions)

Reply:
yes but prio is getting the test to pass.